source/gloperate/include/gloperate/rendering/CameraUtils.h

```diff
@@ -0,0 +1,41 @@
+
+#pragma once
+
+
+#include <glm/mat4x4.hpp>
+
+#include <gloperate/gloperate_api.h>
+
+
+namespace gloperate
+{
+
+/**
+*  @brief
+*    Compute orthographic projection matrix from perspective parameters
+*
+*  @param[in] fovy, aspectRatio, zNear, zFar
+*    same as for glm::perspective
+*
+* @param[in] syncDist
```

Comment:
And please lets exercise some care in the style and lets try to make it consistent (in this case: two spaces before param, just as in the line above)